_includes/metadata.html

```diff
@@ -0,0 +1,142 @@
+<!-- Standard metadata -->
+{%- if site.enable_google_verification -%}
+<meta name="google-site-verification" content="{{ site.google_site_verification }}" />
+{%- endif -%}
+{%- if site.enable_bing_verification -%}
+<meta name="msvalidate.01" content="{{ site.bing_site_verification }}" />
+{%- endif -%}
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>
+{%- if site.title == "blank" -%}
+{%- capture title -%}{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}{%- endcapture -%}
+{%- else -%}
+{%- capture title -%}{{ site.title }}{%- endcapture -%}
+{%- endif -%}
+{%- if page.title != "blank" and page.url != "/" -%}
+{{ title }} | {{ page.title }}
+{%- else -%}
+{{ title }}
+{%- endif -%}
```

Comment:
do you mind correctly indenting this block of code with 4 spaces (no tabs) for better readability? (same request for the code below)

Reply:
Solved in https://github.com/alshedivat/al-folio/pull/481/commits/9648f1bcfff8c0f698118436f1a4c8f5e8f8d518

Source code example with full data:
<img width="1352" alt="Screenshot of Google Chrome (22-12-21, 09-46-06)" src="https://user-images.githubusercontent.com/28966312/147064567-3a1f5cbe-e0ef-4b46-bfd3-0ce02fc73614.png">